src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerTemplate.java

```diff
@@ -133,17 +135,19 @@ public void setCommand(String command) {
         this.command = command;
     }
 
+    @CheckForNull
```

Comment:
Uh, why?

Reply:
Habit I guess. I guess with the other change it's not necessary.

Reply:
In fact it is wrong: if you are going to mark it at all, it should be `@Nonnull` (or `@NonNull` depending on the package imported).